.devcontainer/devcontainer.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "HACS Documentation",
-	"image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:0-14",
+	"image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:16",
```

Comment:
18 is now LTS, why do 16 instead of 18?

Reply:
Will fix your requested changes tomorrow.